helm/draughtsman-chart/templates/network-policy.yaml

```diff
@@ -0,0 +1,18 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  namespace: {{ .Values.resource.default.namespace }}
+  name: {{ tpl .Values.resource.default.name . }}
+spec:
+  podSelector:
+    matchLabels:
+      app: {{ tpl .Values.resource.default.name . }}
+  ingress:
+  - ports:
+    - port: {{ .Values.ports.draughtsman.name }}
+      protocol: {{ .Values.ports.draughtsman.protocol }}
+  egress:
+  - {}  
```

Comment:
policy itself is fine. But some mixed changes make this a bit confusing

Reply:
yeah i did think that -  would you prefer if i removed the templating for now and then add it in a separate PR? There's more to do throughout the manifests too.

Reply:
I think it is fine. Just drop `user` part and add it where it is actually used.